tuxemon/states/combat/combat.py

```diff
@@ -503,40 +503,42 @@ def transition_phase(self, phase: CombatPhase) -> None:
             pass
 
         elif phase == "ran away":
-            self.players[0].set_party_status()
+            pass
```

Comment:
I'm wondering if there would not be a way to get rid of these empty branches... They take some space for nothing.

Maybe we could at least merge them?

Like `elif phase in ("resolve match", "ran away"): pass`, and this could be right before the `else` part to not annoy the reader with it.

Reply:
I changed like in line 442:
```
        elif phase == "resolve match" or phase == "ran away":
            pass
```